### ISSUE TITLE
Expand Fossil Notes 80 char line to 150 char to capture un-truncated items 

### ISSUE THREAD

**Issue by LeannePyle** [bcgov/BCHeritage#776]
To be single line w/ max 150 characters
- [x] Update field
- [x] Update loads

**Comment by bferguso** [comment]
@LeannePyle  - I thought we'd decided to leave it as a single line but to make the max length longer? (120 chars?) Or did that change?

**Comment by LeannePyle** [comment]
Yes we can keep one line, but need to make the accepted length longer - can we do 150 char?

**Comment by bferguso** [comment]
Automated cleanup
